### DISCUSSION TITLE
Inverse Kinematics in C++

### DISCUSSION THREAD

**Discussion by aadhithya-fauna** [NVlabs/curobo#439] [Research Q&A]
Can we use the Inverse kinematics in C++? I don't see much things in the documentation to use the library in C++.

**Comment by balakumar-s** [comment]
cuRobo uses pyTorch to build the compute graph and this is implemented in python. I am not sure if this can be wrapped into a C++ binary.

The easiest way to get access in C++ would be to use a message parsing library to interface between the python code and C++.
